Library/ViewModels/RewardAddOnSelectionViewModel.swift

```diff
@@ -97,6 +97,7 @@ public final class RewardAddOnSelectionViewModel: RewardAddOnSelectionViewModelT
     let projectEvent = slugAndShippingRule.switchMap { slug, shippingRule in
       AppEnvironment.current.apiService.fetchRewardAddOnsSelectionViewRewards(
         slug: slug,
+        shippingEnabled: shippingRule?.location.graphID != nil,
```

Comment:
@singhhari are there ever legitimate situations where we would want to pass `nil` and fetch all locations?

Reply:
The only thing I can think of is the pledge view. We do need to render the entire list of available shipping locations there but I don't think there is a practical use case for ALL locations.

Reply:
Ah right, but that should rely on the regular shipping views that aren't fetched from the expanded list.